tests/unit/core/core.js

```diff
@@ -138,4 +138,89 @@ test( "uniqueId / removeUniqueId", function() {
 	equal( el.attr( "id" ), null, "unique id has been removed from element" );
 });
 
+// Support: Core 1.9 Only, IE8 Only
+// The use of $.trim() in the two tests below this is to account for IE8 missing string.trim()
```

Comment:
Move the support comment to where the support actually exists, not all the way at the top of the function.